plugin/storage/factory_config.go

```diff
@@ -29,13 +30,21 @@ const (
 	DependencyStorageTypeEnvVar = "DEPENDENCY_STORAGE_TYPE"
 
 	spanStorageFlag = "--span-storage.type"
+
+	// DownSamplingRatio is the name of the env var that defines the ratio of spans we would drop
+	DownSamplingRatio = "DOWN_SAMPLING_RATIO"
```

Comment:
the name / description is not clear. Does the value of 0.1 mean we _drop_ 10% or that we _keep_ 10%?

Reply:
NB: I would suggest to be consistent with the regular probabilistic sampler here, i.e. 0.1 means we only _keep_ 10%. But the comment & CLI flag descriptions should be more clear.